CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ This project also adheres to [Semantic Versioning](https://semver.org/spec/v2.0.
 
 Go to [legend](#legend---types-of-changes) for further information about the types of changes.
 
-## [0.0.0.4] - 2023-10-??
+## [1.0.0] - 2023-10-??
```

Comment:
I still propose to not make it 1.0.0
maybe 0.9.0
thus we can do few  0.9.x version

Reply:
Okay. When I went to compare 0.0.0.3 version with Head, it showed 1.0.0, so I assumed that was you intentions.

Reply:
As I said we are close to 1.0.0
I think we should enclose some stage thus 0.9.0 should be released.
Then we need some reflection and feedback from community.
Then we can finall review and version 1.0.0 will be happy to be released.

Reply:
Ya, that's a good idea. 
So far, at least on the AutoIt forum, there has been well over 100 downloads, but no one comments. .... either everyone isn't having any issues or no one can figure out how it works :)

Reply:
Feedback will be when we publish this version, because as so far we are in huge developement process.
And all interested persons are just staying away in order to estabilish of this process.
At lest I think so.